build.gradle

```
@@ -134,6 +134,9 @@ dependencies {
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
 
+    implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '10.1.13'
+    implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '10.1.13'
```

you can do this by dependency management , we do not need direct dependency.

---

under dependencyManagement

```suggestion
    //CVE-2023-41080
    dependencySet(group: 'org.apache.tomcat.embed', version: '10.1.13') {
      entry 'tomcat-embed-core'
      entry 'tomcat-embed-websocket'
    }
```